cf/data/mixin/deprecations.py

```diff
@@ -731,7 +748,11 @@ def partition_boundaries(self):
 
         """
         _DEPRECATION_ERROR_METHOD(
-            "TODODASKMSG - consider using 'chunks' instead"
+            self,
+            "partition_boundaries",
+            message="Consider using the 'chunks' attribute instead.",
```

Comment:
Perhaps, to use the same tone/format as (most, codebase-wide) of the other messages, just:

```suggestion
            message="Use the 'chunks' attribute instead.",
```

?

Reply:
Hi Sadie, I see what you mean, but I try to use "consider" when the suggested alternative doesn't quite do what the deprecated method did, i.e. you can't just change the name and carry on!

Reply:
Ah, that makes sense! Noted for future. Thanks, marking as resolved.